src/constraint_functions.jl

```diff
@@ -478,15 +478,39 @@ function constraints_level_scp(m, n::Storage, per, modeltype::EnergyModel)
     return nothing
 end
 """
-    constraints_level_scp(m, n::Storage{CyclicRepresentative}, per, modeltype::EnergyModel)
+    constraints_level_scp(m, n::Storage{<:Cyclic}, per::TS.AbstractStrategicPeriod, modeltype::EnergyModel)
+
+When a Storage{<:Cyclic} is used in a time structure without `RepresentativeProfile`, the
+final level in an operational scenario is constrained to be the same in all operational
+scenarios for the cyclic constraints.
+"""
+function constraints_level_scp(
+    m,
+    n::Storage{<:Cyclic},
+    per::TS.AbstractStrategicPeriod,
+    modeltype::EnergyModel,
+)
+    # Declaration of the required subsets
+    𝒯ˢᶜ = opscenarios(per)
+    last_scp = [last(t_scp) for t_scp ∈ 𝒯ˢᶜ]
+
+    # Constraint that level is similar to the level in the first scenario
+    for t ∈ last_scp[2:end]
```

Comment:
This looks correct, but is a bit different from the other modes I think which require the sum of deltas to be zero, but would also require setting the level at first or last periods in each scenario to be the same.